src/components/UniversalNavbarExpanded/SampleContent.js

```diff
@@ -1,3 +1,21 @@
+import React from 'react'
+
+const playIcon = () => (
+  <svg
+    width="21"
+    height="21"
+    viewBox="0 0 21 21"
+    fill="none"
+    xmlns="http://www.w3.org/2000/svg"
+  >
+    <path
+      d="M8.39941 5L15.1996 10.8002L8.39941 16.6005V10.8002L8.39941 5Z"
+      fill="#272727"
+    />
+    <circle cx="10.5" cy="10.5" r="10" stroke="black" />
+  </svg>
+)
+
```

Comment:
Just an example of the new icon that will be implemented on CMS, put here for testing purposes locally with `yarn styleguide`.